src/assets/translations/ar.json

```diff
@@ -0,0 +1,177 @@
+{
+  "Arabic": "عربي",
+  "English": "English",
+  "Dashboard": "Dashboard",
```

Comment:
If we're going to provide some arabic translations out of the box, I think we should actually have the valid translations.

Reply:
Yeah, sure in next release maybe

Reply:
Let's remove unused translation files.